extensions/TrackingFlags/Extension.pm

```diff
@@ -335,6 +367,68 @@ sub active_custom_fields {
   @$$fields = sort { $a->sortkey <=> $b->sortkey } values %field_hash;
 }
 
+sub _get_product_version {
```

Comment:
Storing this in the request cache is probably fine. But if we want it in memcached it would need its own (to borrow a term from the windows registry) hive to live in.

It appears https://product-details.mozilla.org/1.0/ is behind cloudfront and returns a response in 200s. Do we really need memcaching for this?

Reply:
Also, the max-age of that response is 300 seconds.

Reply:
Since the [Crash Stats table](https://github.com/mozilla-bteam/bmo/blob/master/extensions/BMO/web/js/firefox-crash-table.js) is also using `firefox_versions.json`, and the version info is not updated so frequently, we should probably cache it for a certain period of time and [try to use it](https://bugzilla.mozilla.org/show_bug.cgi?id=1543489).

I’ll update this to remove the cron dependency and use `Mojo::UserAgent::Role::Cache`.